docs/dev/wandb-service-user.md

```diff
@@ -1,25 +1,64 @@
-# wandb service
+<p align="center">
+  <img src="../../.github/wb-logo-lightbg.png#gh-light-mode-only" width="600" alt="Weights & Biases"/>
+  <img src="../../.github/wb-logo-darkbg.png#gh-dark-mode-only" width="600" alt="Weights & Biases"/>
+</p>
 
-The wandb service is still experimental. It can be enabled by adding the following to your script:
+# `wandb` service
 
-```python
-wandb.require(experiment="service")
-```
+We are excited :tada: to share with you a new feature we call `service`. This feature is still in the experimental stage, but will be integrated by default in `wandb` in the near future.
 
 ## Why would you use this feature?
 
-`wandb-service` improves wandb's handling of multiprocessing and thus improves reliability in a distributed training setting.
+`service` improves `wandb`'s handling of multiprocessing and thus improves reliability in a distributed training setting.
 If you are using `wandb` in a distributed training setup and experiencing hangs, please consider trying out this new feature.
 
-## Installation
+## Usage
+
+### PyTorch Lightning
 
-If you are using `Pytorch-Lightning` please also install this custom branch:
+If you are using `PyTorch Lightning` please:
+
+- Install the most recent version of `wandb` and update `PyTorch Lightning` from the master branch:
 
 ```bash
-pip install --force git+https://github.com/wandb/pytorch-lightning.git@wandb-service-attach
+pip install --upgrade wandb
+pip install --upgrade git+https://github.com/PytorchLightning/pytorch-lightning.git
 ```
 
-(This branch will be upstreamed and be part of the regular Pytorch Lightning package.)
+(The feature support will be part of `PyTorch Lightning`'s [1.6.0 release](https://github.com/PyTorchLightning/pytorch-lightning/pull/11650).)
+
+That's it... no need to change anything in else...
+
+### Others
+
+`service` can be enabled by adding the following to your script:
+
+```python
+def __name__ == "__main__":
+    wandb.require(experiment="service")
+    # <rest-of-your-script-goes-here>
+```
+
+### Advance example:
```

Comment:
```suggestion
### Advanced usage
```

Reply:
maybe advanced usage example? @dmitryduev what do you think? since it is just a simple example?

Reply:
no strong preference to either

Reply:
+1 to "advanced usage example"